### ISSUE TITLE
Get-CRMConnetion throws Method Not Found error

### ISSUE THREAD

**Issue by NixonFin** [seanmcne/Microsoft.Xrm.Data.PowerShell#367]
Hi,

I noticed that there is related issue opened on Mar 29, 2018, but no resolution on how to run this on CRM server as it throws Method Not Found error.

I have freshly installed this module (v. 2.8.1.3) and no previous versions have been installed on the server, so I don't think that problem is in GAC-files. 
Any ideas on how to resolve this as I need to run scripts from CRM server.

Server is 2012 R2 
Powershell version is 5.1

**Comment by seanmcne** [comment]
In the past this has been caused by old gac’d Dlls that conflict. Since CRM server does not require any CRM dlls to be gac’d, can you try using gac util or viewing the gac to ensure no CRM or xrm dlls are in there? Also which command are you using that’s throwing a method not found?

**Comment by seanmcne** [comment]
I think we've chased down the cause of this to .NET 4.8.  I tested a preview build a couple of weeks back that seems to resolve it.  Looking for it to ship and then will update the module.  

For now the workaround is one of the three following: 
1. Connect the first time, encounter the error, then attempt to connect one more time. 
OR 
2. Go to add windows features, under  .NET 4.8 go to WCF services and add http activation. 
OR 
3. Connect using _connect-Crmonline -serverurl org.crm.dynamics.com -forceOAuth 

![image](https://user-images.githubusercontent.com/2292260/61339916-9c2cdf00-a805-11e9-99e6-21f3eafd21db.png)
